src/Serialization/containers.jl

```diff
@@ -3,10 +3,21 @@
 
 @registerSerializationType(Vector)
 
-function save_internal(s::SerializerState, vec::Vector)
-    return Dict(
+is_basic_serialization_type(::Type) = false
```

Comment:
As with previous iterations of this PR: some documentation would be helpful. I.e. here a comment or even a docstring which says what `is_basic_serialization_type` is about / how to use it properly...

Reply:
If you want to get this merged quickly, at least add a FIXME comment indicating that this needs to be documented

Reply:
I added a docstring, but do we want to export this function?